src/test/java/org/broadinstitute/hellbender/tools/copynumber/utils/germlinetagging/SimpleGermlineTaggerUnitTest.java

```diff
@@ -0,0 +1,152 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.germlinetagging;
+
+import com.google.common.collect.ImmutableSortedMap;
+import com.google.common.collect.Maps;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.AnnotatedInterval;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceUtils;
+import org.spark_project.guava.collect.Lists;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.List;
+
+public class SimpleGermlineTaggerUnitTest extends GATKBaseTest {
+    private static final String REF = hg19MiniReference;
+    public static final String TEST_GERMLINE_TAGGING_ANNOTATION = "germline_tagging";
+
+    @Test(dataProvider = "simpleTests")
+    public void testSimpleTagging(List<AnnotatedInterval> tumorSegments, List<AnnotatedInterval> normalSegments,
+                                  List<AnnotatedInterval> gt) {
+
+        final List<AnnotatedInterval> testResult = SimpleGermlineTagger.tagTumorSegmentsWithGermlineActivity(tumorSegments,
+                normalSegments, "call",
+                ReferenceUtils.loadFastaDictionary(new File(ReferenceUtils.getFastaDictionaryFileName(REF))), TEST_GERMLINE_TAGGING_ANNOTATION, 10);
+
+        Assert.assertEquals(testResult, gt);
+    }
+
+    @DataProvider(name = "simpleTests")
+    public Object [][] createSimpleTests() {
+        return new Object[][] {
+                {
+                        // Trivial case
+                        Lists.newArrayList(
```

Comment:
Why not `Arrays.asList()`? Do these need to be modifiable? Also, you could shorten the data provider by putting the array -> list conversion into the test method.

Reply:
Converted to `Arrays.asList()`.  Done.

No action on the array -> list conversion in the test.